docs/src/components/landingBlocks/BuiltByBlock.tsx

```diff
@@ -5,10 +5,10 @@ import NomicFoundationLogo from "../../assets/images/nomic-foundation-logo.png";
 
 import Section from "../Section";
 import { appTheme } from "../../themes";
-import defaultProps from "../ui/default-props";
+import homepageContent from "../../content/home";
 
 interface Props {
-  content: typeof defaultProps.defaultBuiltByBlockContent;
+  content: typeof homepageContent.builtByBlockContent;
```

Comment:
are we importing `homepageContent` only to get a type? let's switch to a explicit type declaration. Not in this PR. Please create a task in Trello